org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/semanticTokens/SemanticHighlightReconcilerStrategy.java

```diff
@@ -190,7 +193,7 @@ private void saveStyle(final Pair<@Nullable SemanticTokens, @Nullable SemanticTo
 		final var semanticTokensDataStreamProcessor = this.semanticTokensDataStreamProcessor;
 		final var styleRangeHolder = this.styleRangeHolder;
 		if (!dataStream.isEmpty() && semanticTokensDataStreamProcessor != null && styleRangeHolder != null) {
-			List<StyleRange> styleRanges = semanticTokensDataStreamProcessor.getStyleRanges(dataStream,
+			List<StyleRange> styleRanges = semanticTokensDataStreamProcessor.getTokens(dataStream,
```

Comment:
to call `getTokens` and to save it into a variable of type `StyleRange` is a bit awkward, but I do not have a suggestion for a better name at this moment, do you have a better idea?

Reply:
`getVisualTokens()`?

Reply:
since it is an `...TokensDataStreamProcessor`, maybe `getTokensData`?

After that I would be happy to merge.

Reply:
Sounds good to me - made this change.